pom.xml

```diff
@@ -0,0 +1,198 @@
+<!--
+  ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.wso2</groupId>
+        <artifactId>wso2</artifactId>
+        <version>1.2</version>
+    </parent>
+
+    <groupId>io.asgardio.tomcat.oidc.agent</groupId>
+    <modules>
+        <module>io.asgardio.tomcat.oidc.agent</module>
+        <module>io.asgardio.tomcat.oidc.sample</module>
+    </modules>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>tomcat-oidc-agent</artifactId>
+    <version>0.1.1-SNAPSHOT</version>
```

Comment:
Need to start with 0.1.0-SNAPSHOT

Reply:
Resolved with https://github.com/asgardio/asgardio-tomcat-oidc-agent/pull/1/commits/6b6ec648bf4aa46847e8c22e02fe4500e8a7d18e.